### ISSUE TITLE
Generate different image sizes and support the "srcset" img tag attribute

### ISSUE THREAD

**Issue by lairdshaw** [mybbgroup/Image-auto-resizer#5]
As suggested by WallBB [here](https://community.mybb.com/thread-230646-post-1359997.html#pid1359997).

**Comment by Eldenroot** [comment]
Not a bad idea, but this will not save space, image size will be much bigger in total because we will generate multiple same imgaes with different widthxheight, right?

**Comment by lairdshaw** [comment]
Yes, exactly right - but I figure that the name of the plugin only indicates that it "resizes" images, so potentially it could be dual-purpose: resize images to minimise disk usage, or resize images to provide multiple variants for browser compatibility via the `srcset` attribute.

I'm open to arguments though that this would over-complicate the plugin and should be done in a separate plugin - plus, I can't see myself implementing the feature any time soon (am happy for anybody else to take it on in the meantime).

**Comment by Eldenroot** [comment]
I agree with you... there are another issues and feature request with much higher priority. Anyway if anybody will send a PR, I will be happy for that and I have no problem to add it into the plugin.
